metricbeat/module/aerospike/aerospike.go

```diff
@@ -22,9 +22,37 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/elastic/elastic-agent-libs/transport/tlscommon"
+
 	as "github.com/aerospike/aerospike-client-go"
 )
 
+type Config struct {
+	ClusterName *string           `config:"cluster_name"`
```

Comment:
Do we need to use a pointer here for `ClusterName`? `string` would be cleaner. Also, is `ClusterName` somehow required when using TLS?

Reply:
As commented here https://github.com/elastic/beats/pull/38126#discussion_r1555338419 this setting is not actually required to make TLS work. It's only needed to avoid that a client connects to a different/unexpected cluster. 

I think we can get rid of the setting, if we want to make the PR only focus on TLS